corehq/apps/locations/views.py

```diff
@@ -122,6 +123,7 @@ class LocationsListView(BaseLocationView):
 
     @use_jquery_ui
     @location_safe
+    @method_decorator(require_can_edit_commcare_users)
```

Comment:
@sheelio are you referring to the "Users" tab of an "Edit Location" page?  If so, I believe this check needs to go at the top of this method: https://github.com/dimagi/commcare-hq/blob/f806fa5e06fd86dc999ecdefec2cf3621da555f0/corehq/apps/locations/views.py#L602-L615